### ISSUE TITLE
Can't run olex2 using wayland

### ISSUE THREAD

**Issue by MilitaoLucas** [pcxod/olex2#6]
I was trying for a while to get olex2 working on Wayland on Fedora. But I wasn't able to. I tried to run it under xWayland using:
```
export GDK_BACKEND=x11
```
But I always get:
```
./src/unix/glx11.cpp(626): assert ""Assert failure"" failed in InitVisual(): Failed to get a XVisualInfo for the requested attributes.
````
I tried to compile it from source to see if it would work, but I couldn't. I had some of the same problems of the Manjaro issue, but couldn't get scons to work after following the recommendations. 
I awlays get:
```
/usr/bin/ld: build/scons/gnu/release-64bit/py3.1/olex/xpprocs.o: undefined reference to symbol 'FcConfigAppFontAddDir'
/usr/bin/ld: /usr/lib64/libfontconfig.so.1: error adding symbols: DSO missing from command line
```

**Comment by MilitaoLucas** [comment]
I got it working by compiling from source. If you accept PRs, I can organize my work and create a README with instructions on how to compile from source. The gist is that using wxWidgets 3.0.5 solved most of my problems. I couldn't get it working natively in Wayland for now, but maybe when I got the time I will work on it. I think that somehow compiling in a machine running in Wayland solved the problem.

**Comment by pcxod** [comment]
Yeah, there was that problem with the context selection. I think it has been solved in the DEV version:
http://www2.olex2.org/olex2-distro/1.5-dev/

You can give it a try. If you want to contribute into the manual - please do!

**Comment by MilitaoLucas** [comment]
I think I will probably elaborate a section on compiling from source, as the Wayland problem was already fixed in the new version. Also, I was thinking of creating this as a README on this repository, as it would be easier to see. But I could do it in the Documentation repo. Or could do both.
